docs/docsgen/source/expect_onnxruntime.rst

```diff
@@ -0,0 +1,69 @@
+.. _l-function-expect:
+
+Sample operator test code
```

Comment:
This section feels out of place in the "operator" documentation. I am trying to understand it's purpose.

The code itself seems useful as a utility, but not relevant to this "operator" chapter (or whatever it is called).

Of course, it is useful to explain to the reader how to interpret the `expect(...)` line found in the documentation. That explanation seems appropriate for this section, but would be better explained in text on what the `expect` API means.

Reply:
This comes from a comment from Marjan. He was telling that none of the examples in the documentation of every operator can be run. Two reasons, missing imports (added) and a function expect difficult to find. So I suggested to give an implementation of this function based on onnxruntime as a way for the user to check the output against the expected outputs.

Reply:
I see. Ok. I think the fundamental issue is the code is being used for two purposes: to generate test-cases in proto format, but also to serve as example (input, output) pairs to be included in the document. IMHO, what is most useful in the documentation is something that can be *read*: that is, sample input and sample output. But we currently don't do a great job in the documentation part (since we just copy the test-generation code). Anyway: that is a separate issue, maybe a future potential improvement.